### PR TITLE
Update UG on undo, redo, help, exit commands

### LINKED ISSUE #198: Update UG on redo and undo feature

### PR DESCRIPTION
**Pull request by lightz96**
Resolves #198

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -253,52 +254,65 @@ NOTE: Assume entry at index 1 is `[-] Laksa ($10.50) Date: 18/10/2019 Remarks: U
...
+* `<<CloneTag, clone>>`
+* `<<TagTag, tag>>`
+* `<<UntagTag, untag>>`
+* `<<BudgetTag, budget>>`
 
-* `add_income n/Allowance v/1000 t/Monthly` +
-  `list` +
-  `undo`
+Examples: +
+Suppose you want to delete the second transaction (Bursary) in THRIFT, you remove the third transaction instead!
```
Comment (lye-jw) [inline docs/UserGuide.adoc]: Would a "but" in between sentences make them flow better? e.g. "Suppose you want to delete the second transaction (Bursary) in THRIFT, **but** you remove the third transaction instead!"

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -253,52 +254,65 @@ NOTE: Assume entry at index 1 is `[-] Laksa ($10.50) Date: 18/10/2019 Remarks: U
 
 [[UndoTag]]
 === Undoing the previous undoable command: `undo`
+Do not panic if you have accidentally made a mistake in THRIFT, like deleting the wrong transaction.
+THRIFT provides an `undo` command to save your trouble! You can revert to the previous state
```
Comment (lye-jw) [inline docs/UserGuide.adoc]: "**solve** your trouble" may be more grammatically correct.

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -253,52 +254,65 @@ NOTE: Assume entry at index 1 is `[-] Laksa ($10.50) Date: 18/10/2019 Remarks: U
...
+* `<<UntagTag, untag>>`
+* `<<BudgetTag, budget>>`
 
-* `add_income n/Allowance v/1000 t/Monthly` +
-  `list` +
-  `undo`
+Examples: +
+Suppose you want to delete the second transaction (Bursary) in THRIFT, you remove the third transaction instead!
+One possible solution is that you add the transaction manually back into THRIFT, but this is too troublesome! You can
+retrieve back the deleted transaction by performing `undo`.
```
Comment (lye-jw) [inline docs/UserGuide.adoc]: "retrieve" should be enough instead of "retrieve back".

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -253,52 +254,65 @@ NOTE: Assume entry at index 1 is `[-] Laksa ($10.50) Date: 18/10/2019 Remarks: U
...
+. You will see a success message displays in the result box, and the transaction (Chicken Rice)
+adds back to its original position in the list:
++
+image::UndoAndRedo/undocommandresult.PNG[, 600]
 
 [[RedoTag]]
 === Redoing the undone command: `redo`
-
-Redo the <<Undoable, undoable command>> that was previously undone by the user.
+Overuse of `undo` command? No worries, THRIFT provides `redo` command to help you in this situation.
```
Comment (lye-jw) [inline docs/UserGuide.adoc]: "**Misuse**" could convey the meaning more clearly than "overuse".

**Review by lye-jw — CHANGES_REQUESTED** [review]
LGTM after some quick changes!

**Comment by lightz96** [comment]
Merge PR #210 first

**Review by lye-jw — APPROVED** [review]

**Pull request merged**